zellij-server/src/os_input_output.rs

```diff
@@ -300,6 +300,7 @@ fn spawn_terminal(
                     || command.ends_with("emacs")
                     || command.ends_with("nano")
                     || command.ends_with("kak")
+                    || command.ends_with("hx")
```

Comment:
Could you also add `helix`? Some distros use the `helix` command instead of the `hx` command e.g. ArchLinux and openSUSE

Reply:
Added in https://github.com/zellij-org/zellij/pull/2156/commits/9acf464e566643863f13fcc294db032349395888